MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Indirect/Elwin.h

```diff
@@ -34,10 +34,12 @@ private slots:
   void unGroupInput(bool error);
   void saveClicked();
   void plotClicked();
+  void plotCurrentPreview();
 
 private:
   Ui::Elwin m_uiForm;
   QtTreePropertyBrowser *m_elwTree;
+  Mantid::API::MatrixWorkspace_sptr m_ElInputWS;
```

Comment:
This is consistent across Indirect interfaces currently - this perhaps need a separate issue for cleaning up.